packages/fxa-auth-server/test/local/sentry.js

```diff
@@ -10,12 +10,14 @@ const sinon = require('sinon');
 const verror = require('verror');
 const Hapi = require('@hapi/hapi');
 const Sentry = require('@sentry/node');
-const error = require('../../lib/error');
+const AppError = require('../../lib/error');
```

Comment:
I refactored this, because when I revisited this code calling it 'error' seemed a bit misleading and ambiguous.